packages/web/webpack/constants.js

```diff
@@ -6,7 +6,6 @@ const devServer = {
 const Env = {
     DEVELOPMENT: "dev",
     PRODUCTION: "production",
-    STAGE: "staging",
```

Comment:
This is intentional?  We never want staging?

Reply:
Yep--there's no staging deployment for this tool, because it's not really "deployed" as such. It's "distributed." The language here could maybe be updated if it's confusing, but there are just two types of builds, "dev" builds and "non-dev" builds. The "non-dev" builds are what are distributed, so, I figured it's fair to continue to call them "production" builds. Let me know if you think otherwise.

Reply:
This change I could bring back--I was trying to keep `web` and `desktop` config more or less the same, and while it doesn't make sense for `desktop` to have a "staging" (because it isn't deployed, it's distributed, so there's only "dev" and "not dev"), it would make sense for the `web` version of this to have a "staging." All said, the `web` version of this tool has never been deployed anywhere.